mobile/src/viewModels/VersionViewModel.ts

```diff
@@ -0,0 +1,28 @@
+import { VersionStub } from '../../../common/controllers/VersionController';
+import AppController from 'src/controllers';
+
+const SUPPORTED_DB_VERSION = '1.0.3';
```

Comment:
Can we move this to the `.env` file and use that env variable here instead? (or is this done through the template file already? if so, can we put this version as 1.0 instead)

Reply:
Yeah this file is generated from the template, ill set to 1.0